go.mod

```diff
@@ -0,0 +1,21 @@
+module sqlflow.org/sqlflow
+
+require (
+	github.com/apache/thrift v0.13.0 // indirect
```

Comment:
Is there any mechanism of go mod that could handle such indirect modules automatically?

Reply:
Please check out the document link above. `go.mod` and `go.sum` are generated automatically.